### DISCUSSION TITLE
System-upgrade unable to pull latest image

### DISCUSSION THREAD

**Discussion by ilbarone87** [rancher/rke2#6279] [Q&A] [answered]
Hello everyone, I've found my system-upgrade job failed this morning due to this error:

"Failed to pull image "rancher/rke2-upgrade:v1.28.11-rke2r1": rpc error:
  code = NotFound desc = failed to pull and unpack image "docker.io/rancher/rke2-upgrade:v1.28.11-rke2r1":
  failed to resolve reference "docker.io/rancher/rke2-upgrade:v1.28.11-rke2r1": docker.io/rancher/rke2-upgrade:v1.28.11-rke2r1"
The vm has connectivity.

Is there anyone experiencing the same?

EDIT: just noticed that image is missing still in docker hub.

**Comment by iFloris** [comment]
Yes, I am also seeing this behaviour

**Comment by brandond** [comment, marked as answer]
There was a failure in tag CI for the rke2-upgrade repo. CI has been re-run and the images should now be available.

**Reply by ilbarone87** [reply]
Can confirm all is fixed now, upgraded successfully
